scripts/automation/commandlint/run.py

```diff
@@ -85,6 +722,13 @@ def dump_no_help(modules):
                                      pkgutil.iter_modules(mods_ns_pkg.__path__)]
     except ImportError:
         pass
-    exit_value = dump_no_help(installed_command_modules)
-    sys.exit(exit_value)
 
+    result, failed_commands = dump_no_help(installed_command_modules)
+
+    if failed_commands or result != 0:
+        print('==== FAILED COMMANDS ====')
+        print(json.dumps(failed_commands, sort_keys=True, indent=4))
+    else:
+        print('==== ALL COMMANDS PASS! ====')
```

Comment:
We don't need to print anything if the command pass. Let it return zero that will be enough.